deploy/templates/deployment.yaml

```diff
@@ -24,13 +24,37 @@ spec:
       nodeSelector:
         {{- toYaml .Values.nodeSelector | nindent 8 }}
       {{- end }}
-      {{- if .Values.affinity }}
       affinity:
+      {{- if .Values.affinity }}
         {{- toYaml .Values.affinity | nindent 8 }}
+      {{- else }}
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
```

Comment:
`weight` seems to be unnecessary if only one preference

Reply:
This is a required field, range 1-100